kpi/views/tmp_kpi_forecasts.view.lkml

```diff
@@ -0,0 +1,105 @@
+view: tmp_kpi_forecasts {
+  derived_table: {
+    sql:
+      WITH most_recent_forecasts AS (
+        SELECT aggregation_period,
+               metric_alias,
+               metric_hub_app_name,
+               metric_hub_slug,
+               MAX(forecast_predicted_at) AS forecast_predicted_at
+          FROM `moz-fx-data-shared-prod.telemetry_derived.kpi_forecasts_v0`
+         GROUP BY 1, 2, 3, 4
+      )
+      
+      SELECT forecasts.* EXCEPT(forecast_parameters)
+        FROM `moz-fx-data-shared-prod.telemetry_derived.kpi_forecasts_v0` AS forecasts
+        JOIN most_recent_forecasts
```

Comment:
Is this where the observed values come from? What happens when you try to plot the most recent observed values with future values from the forecast, will this join allow for that?

Reply:
The short answer is that it works as expected!

This version of the forecast table includes forecasted and observed values at the time the forecast was generated. So this join returns the most recent versions of the observed and forecasted datasets for all combinations of aggregation_period, metric_alias, metric_hub_app_name, and metric_hub_slug.